packages/api/src/hooks/useAvailableWallets.ts

```diff
@@ -0,0 +1,78 @@
+import React from 'react';
+import useFetch from '../useFetch';
+import useWalletAccountsList from './useWalletAccountsList';
+import useAuthorize from './useAuthorize';
+import useCurrencyConfig from './useCurrencyConfig';
+
+const useAvailableWallets = () => {
+    const { data } = useAuthorize();
+    const { data: account_type_data, ...rest } = useFetch('get_account_types', {
```

Comment:
@thisyahlen-deriv we already have `useAvailableAccounts` hook which passes all wallets for each landing_company. We need to get the wallets for each different landing_compay, compare it with the added_wallets and sort the rest which are remaining